spec/codecs/cef_spec.rb

```diff
@@ -452,7 +452,7 @@ def validate(e)
       end
     end
 
-    let (:trim_additional_fields_with_dot_notations) {'CEF:0|security|threatmanager|1.0|100|trojan successfully stopped|10|src=10.0.0.192 dst=12.121.122.82 ad.field[0]=field0 ad.name[1]=new_name'}
+    let (:trim_additional_fields_with_dot_notations) {'CEF:0|security|threatmanager|1.0|100|trojan successfully stopped|10|src=10.0.0.192 ad.Authentification=MICROSOFT_AUTHENTICATION_PACKAGE_V1_0 ad.Error_,Code=3221225578 dst=12.121.122.82 ad.field[0]=field0 ad.name[1]=new_name'}
```

Comment:
You change the content of the test event, but you did not change the `insist` on line 461 and 462.

Also I am not sure, if it is the correct behavior to drop fields with `.` (dots) in the field name. Sure, in ES fields with `.` in the field name could lead to problems, but ES is not the only target in Logstash.